internal/cli/local/status/status.go

```diff
@@ -0,0 +1,107 @@
+package status
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	"github.com/weka/gohomecli/internal/cli/app/hooks"
+	"github.com/weka/gohomecli/internal/local/chart"
+	"github.com/weka/gohomecli/internal/utils"
+)
+
+var Cli hooks.Cli
+
+var statusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Get the status of Weka Home components",
+	Long:  "Get the status of Weka Home components, including pods and the entire Weka Home instance",
+}
+
+var podsCmd = &cobra.Command{
+	Use:   "pods",
+	Short: "Check the status of pods",
+	Long:  fmt.Sprintf("Check the status of pods and get non-running ones in the %s namespace", chart.ReleaseNamespace),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		detailed, _ := cmd.Flags().GetBool("detailed")
+		outputFormat, _ := cmd.Flags().GetString("output")
+
+		if !detailed && cmd.Flags().Changed("output") {
+			utils.UserError("The --output flag can only be used with the --detailed flag.")
+		}
+
+		pods, err := chart.GetNonRuninngPods()
+		if err != nil {
+			utils.UserError(err.Error())
+		}
+		if len(pods) == 0 {
+			utils.UserNote("All pods are running.")
+			return nil
+		}
+		if detailed {
```

Comment:
very nested logic, i suggest to extract few functions to make it more readable and less complex